doc/conf.py

```diff
@@ -341,10 +341,15 @@ def setup(app):
     'find_mayavi_figures': find_mayavi_figures,
     'expected_failing_examples': ['../examples/no_output/plot_raise.py',
                                   '../examples/no_output/plot_syntaxerror.py'],
-    'binder': {'org': 'sphinx-gallery',
-               'repo': 'sphinx-gallery.github.io',
+    'binder': {#'org': 'sphinx-gallery',
```

Comment:
I will update these back to their respective values when we're +1 on a merge...this is just for testing